medication statements/README.md

```diff
@@ -0,0 +1,450 @@
+
+
+<details>
+    <summary>Example XML</summary>
+
+```
+<MedicationStatement classCode="SBADM" moodCode="ORD">
+   <id root="0FC9E511-9CCA-42DD-986A-D58ABD8208C8"/>
+   <statusCode code="COMPLETE"/>
+   <effectiveTime>
+      <low value="20130103"/><high value="20130602"/>
+   </effectiveTime>
+   <availabilityTime value="20130103"/>
+   <consumable typeCode="CSM">
+      <manufacturedProduct classCode="MANU">
+         <manufacturedMaterial determinerCode="KIND" classCode="MMAT">
+            <code code="178011000001104" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Zimovane LS 3.75mg tablets (Sanofi)">
+              </code>
+         </manufacturedMaterial>
+      </manufacturedProduct>
+   </consumable>
+   <component typeCode="COMP">
+      <ehrSupplyPrescribe>
+         <id root="61B8E9FD-552A-42A1-8562-208B612C429B"/>
+         <code code="394823007" displayName="NHS Prescription" codeSystem="2.16.840.1.113883.2.1.3.2.4.15"/>
+         <statusCode code="COMPLETE"/>
+         <availabilityTime value="20130103"/>
+         <quantity value="150" unit="1">
+            <translation value="150">
+               <originalText>tablet</originalText>
+            </translation>
+         </quantity>
+         <inFulfillmentOf typeCode="FLFS">
+            <priorMedicationRef moodCode="INT">
+               <id root="E41C267A-D303-4A13-8F50-C93A1543A279"/>
+            </priorMedicationRef>
+         </inFulfillmentOf>
+         <pertinentInformation typeCode="PERT">
+            <pertinentSupplyAnnotation>
+               <text>Expected Supply Duration: 150 day</text>
+            </pertinentSupplyAnnotation>
+         </pertinentInformation>
+      </ehrSupplyPrescribe>
+   </component>
+   <pertinentInformation typeCode="PERT">
+      <pertinentMedicationDosage classCode="SBADM" moodCode="RMD">
+         <text>One To Be Taken At Night</text>
+      </pertinentMedicationDosage>
+   </pertinentInformation>
+   <Participant typeCode="AUT" contextControlCode="OP">
+      <agentRef classCode="AGNT">
+         <id root="2D70F602-6BB1-47E0-B2EC-39912A59787D"/>
+      </agentRef>
+   </Participant>
+</MedicationStatement>
+```
+</details>
+
+### Plan
+
+| Mapped to (XML HL7)                                                                                               | Mapped from (JSON FHIR / other source )                                                                                            |
+|-------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------|
+| MedicationStatement \[@classCode]                                                                                 | fixed value = `"SBADM"`                                                                                                            |
```

Comment:
Some of these values are fixed in the HL7 MIM, so they would never be anything else. I brought this up with in the previous review of this table, some have been removed but there are still some present. I'll leave a comment against all the fixed ones (by the MIM).